search-service/src/main/kotlin/com/egm/stellio/search/model/AttributeInstanceResult.kt

```diff
@@ -8,5 +8,6 @@ data class AttributeInstanceResult(
     val instanceId: URI? = null,
     val datasetId: URI? = null,
     val value: Any,
-    val observedAt: ZonedDateTime
+    val observedAt: ZonedDateTime,
+    val payload: String? = null
```

Comment:
I'm wondering if cleaner to have two such data classes: one for normal representation and one for simplified representation.